package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && mocha --timeout 20000"
+    "test": "if-node-version \">=4\" xo && mocha --timeout 20000"
```

Comment:
@SBoudrias which is worse, adding an extra package (if-node-version) and associated complexity? Or having an old version of a devDependency pinned for the foreseeable future?

I don't like old things, but that's just me. :)